frontend/src/store/tagStore.ts

```diff
@@ -82,11 +82,11 @@ export const useTagStore = defineStore('tag', () => {
     }
   }
 
-  async function searchTagging(tagset: Array<Tag> = []) {
+  async function searchTagging(bucketId: string = '', tagset: Array<Tag> = []) {
```

Comment:
bucketId should be the second argument, and probably should not hard default to empty string when unspecified, as that will limit this store action's utility in the long run in the event you need to do a COMS-wide search.